Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -462,36 +468,64 @@ def get_family_info_command(intezer_api: IntezerApi, args: dict) -> CommandResul
 
 # region Enrich DBot
 
-def enrich_dbot_and_display_file_analysis_results(intezer_result):
+def enrich_dbot_and_display_file_analysis_results(intezer_result: dict, file_metadata: dict):
     verdict = intezer_result.get('verdict')
     sha256 = intezer_result.get('sha256')
     analysis_id = intezer_result.get('analysis_id')
+    md5 = file_metadata.get('md5')
+    sha1 = file_metadata.get('sha1')
 
-    dbot = {
-        'Vendor': 'Intezer',
-        'Type': 'hash',
-        'Indicator': sha256,
-        'Score': dbot_score_by_verdict.get(verdict, 0)
-    }
-
-    file = {'SHA256': sha256, 'Metadata': intezer_result, 'ExistsInIntezer': True}
+    dbot_entry, file = _get_dbot_score_and_file_entries(intezer_result, file_metadata)
 
     if verdict == 'malicious':
         file['Malicious'] = {'Vendor': 'Intezer'}
 
+    intezer_result['sha1'] = sha1
+    intezer_result['md5'] = md5
+
     presentable_result = _file_analysis_presentable_code(intezer_result, sha256, verdict)
 
     return CommandResults(
         readable_output=presentable_result,
         raw_response=intezer_result,
         outputs={
-            outputPaths['dbotscore']: dbot,
+            outputPaths['dbotscore']: dbot_entry,
             outputPaths['file']: file,
             'Intezer.Analysis(val.ID && val.ID == obj.ID)': {'ID': analysis_id, 'Status': 'Done'}
         }
     )
 
 
+def _get_dbot_score_and_file_entries(file_analysis_result: dict, file_metadata: dict) -> Tuple[List[dict], dict]:
+    verdict: str = file_analysis_result.get('verdict', '')
+    sha256 = file_metadata.get('sha256')
```

Comment:
Shouldn't you continue to fetch `sha256` from integer_result?

```suggestion
    sha256 = file_analysis_result.get('sha256')
```

Reply:
Both reports return the same value for sha256 and since I'm now getting all the other hashes from the file_metada report, I think it's more clear to get the sha256 from this report as well.